src/executables/srpd_common.c

```diff
@@ -0,0 +1,218 @@
+/**
+ * @file srpd_common.c
+ * @author Ondrej Kusnirik <Ondrej.Kusnirik@cesnet.cz>
+ * @brief common functions for sysrepo-plugind
+ *
+ * @copyright
+ * Copyright (c) 2018 - 2022 Deutsche Telekom AG.
+ * Copyright (c) 2018 - 2022 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define _GNU_SOURCE
+
+#include "srpd_common.h"
+
+#include <errno.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+#include "bin_common.h"
+#include "srpd_aging.h"
+#include "sysrepo.h"
+
+struct srpd_int_plugin_s int_plugins[1] = {{aging_init_cb, aging_cleanup_cb, "aging_plugin"}};
+
+void
+srpd_error_print(int sr_error, const char *format, ...)
+{
+    va_list ap;
+    char msg[2048];
```

Comment:
Avoid fixed buffer length for messages.

Reply:
Error print moved back to sysrepo-plugind.c